docker-compose.yml

```diff
@@ -100,3 +100,14 @@ services:
       - cachito-api
       - nexus
       - rabbitmq
+
+  flower:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-flower
+    environment:
+      RABBITMQ_MNG: cachito:cachito@cachito_rabbitmq_1
+    # Override the default command so that Flower get access to the broker and tasks
+    command: ["./docker/flower-waits-for-cachito.sh"]
```

Comment:
this file does not exist in the image, therefore, it cannot run.

Reply:
it was copied with all other cachito files in [Dockerfile-flower](https://github.com/release-engineering/cachito/pull/362/files#diff-1545cf98a2552ea4cbbe80e18d08a7ce72e3761b1651e19ee6df3fd088887964R12)

Reply:
You are right.

Do we need to copy the whole repository to this image though? Could we copy just the script instead?

Reply:
Also, if we do not need the whole repository to be copied into the image, we can also reduce the context by referencing the `docker` directory instead (this would speed up this build

Reply:
@athos-ribeiro Ok, I copied only `docker` and `tasks` directories.